lib/vmdb/appliance.rb

```diff
@@ -127,7 +127,11 @@ def self.get_build
 
     def self.get_network
       retVal = {}
-      retVal[:hostname] = LinuxAdmin::Hosts.new.hostname
+
+      dns = LinuxAdmin::Dns.new
+      retVal[:hostname]      = LinuxAdmin::Hosts.new.hostname
+      retVal[:primary_dns]   = dns.nameservers[0]
+      retVal[:secondary_dns] = dns.nameservers[1]
```

Comment:
Nope, it doesn't actually shell out at all. It reads the file (in this case /etc/resolv.conf) when the object is instantiated and then makes the nameservers and search order available as `:attr_accessor`s